content/en/contribute/code/core/dev-environment.md

```diff
@@ -139,15 +139,26 @@ curl -X PUT "http://medic:password@localhost:5984/_node/$COUCH_NODE_NAME/_config
 
 #### CouchDB Setup in CHT 4.x
 
-Create a `docker-compose.yml` file under the `couchdb` folder. 
+Create a `docker-compose.yml` and `couchdb-override.yml` files under the `~/cht-docker` folder with this code:
 
-Copy the content of the CouchDB `docker-compose` file from the of the CHT release version you are trying to run locally. For example, this is [the file](https://staging.dev.medicmobile.org/_couch/builds_4/medic%3Amedic%3Amaster/docker-compose%2Fcht-couchdb-clustered.yml) for the most recent build.
+```
+mkdir -p ~/cht-docker
+curl -s -o ~/cht-docker/docker-compose.yml https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:4.1.0/docker-compose/cht-couchdb.yml
+cat > ~/cht-docker/couchdb-override.yml << EOF
+version: '3.9'
+services:
+    couchdb:
+        ports:
+          - "5984:5984"
+          - "5986:5986"
+EOF
+```
 
-Startup CouchDB:
+Now you can start CouchDB. The login for your CHT instance will be `medic` and the `password` will be password:
 
 ```shell
-cd couchdb 
-docker-compose up
+cd ~/cht-docker 
+COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose -f docker-compose.yml -f couchdb-override.yml up -d
```

Comment:
oh - I missed this - sorry!

I tried it and you can't override an already declared element in a the `environment:` stanza, only declare a new one :disappointed: .

That said, I did discover that there's a [default override name](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files) of `docker-compose.override.yml` which means the command could be simplified to

```
COUCHDB_USER=medic COUCHDB_PASSWORD=password docker-compose up -d
```

I think it might be worth doing doc site wide, as we use overrides now on self hosted [multi](https://docs.communityhealthtoolkit.org/apps/guides/hosting/4.x/self-hosting-multiple-nodes/) docs too.  I'll open a ticket!